web/src/app/lists/QueryList.js

```diff
@@ -72,12 +78,16 @@ export default function QueryList(props) {
     pollInterval: POLL_INTERVAL,
   })
 
+  let nodes = []
   let items = []
   let loadMore
+  let hasNextPage = false
 
   if (data && data.data && data.data.nodes) {
+    nodes = data.data.nodes
     items = data.data.nodes.map(props.mapDataNode)
```

Comment:
Can we simplify everything accessing this by just doing something like `const nodes = _.get(data, 'data.nodes', [])` on line 81?

Reply:
We can, but I was avoiding refactoring more code to instead use the if statement already in place without pulling more stuff in

I can make the adjustments though if preferred

Reply:
It seems odd to be accessing the same data in two different ways. I think it would be more clear if we simplify and clean it up